app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/splash/SplashScreen.kt

```diff
@@ -49,13 +49,15 @@ import com.ramcosta.composedestinations.navigation.DestinationsNavigator
 import com.ramcosta.composedestinations.navigation.EmptyDestinationsNavigator
 import kotlinx.coroutines.launch
 
+@Suppress("LongMethod")
 @Destination(start = true)
 @Composable
 fun SplashScreen(navigator: DestinationsNavigator, viewModel: SplashViewModel = hiltViewModel()) {
     val showLoginComponents by viewModel.shouldNavigateToLogin.collectAsState()
     val shouldShowLoading by viewModel.shouldShowLoading.collectAsState()
     val shouldShowError by viewModel.hasError.collectAsState()
     val isLoginSuccess by viewModel.isLoginSuccess.collectAsState()
+    val isAlreadyLoggedIn by viewModel.isAlreadyLoggedIn.collectAsState()
```

Comment:
Why don't we just reuse the `isLoginSuccess` flow for this? It will be one less thing to manage 🤔